test/crs/conversions.jl

```diff
@@ -807,6 +865,18 @@
       @inferred convert(LatLon{WGS84{2296}}, c4)
       @inferred convert(LatLon{WGS84{0}}, c5)
     end
+
+    @testset "LatLonAlt: Datum conversion" begin
+      # TODO
+    end
+
+    @testset "GeocentricLatLon: Datum conversion" begin
+      # TODO
+    end
+
+    @testset "GeocentricLatLonAlt: Datum conversion" begin
+      # TODO
+    end
```

Comment:
@uriele what do you think of adding tests for these cases for completeness? We will look into the added conversion methods more carefully during the day.